capitalize.js

```diff
@@ -0,0 +1,5 @@
+function capitalize(string) {
+  return string.charAt(0).toUpperCase() + string.slice(1);
+}
```

Comment:
You can access elements in the string by index, and you could use an arrow function:

```js
const capitalize = string => string[0].toUpperCase() + string.slice(1)
```